src/interfaces.ts

```diff
@@ -16,14 +16,16 @@ declare module 'aurelia-dependency-injection' {
 
 export type RouteConfigSpecifier = string | RouteConfig | ((instruction: NavigationInstruction) => string | RouteConfig | Promise<string | RouteConfig>);
 
+export type ViewModelSpecifier = () => Function | Promise<Function | Record<string, any>>;
+
 /**
 * A configuration object that describes a route.
 */
 export interface RouteConfig {
   /**
   * The route pattern to match against incoming URL fragments, or an array of patterns.
   */
-  route: string | string[];
+  route?: string | string[];
```

Comment:
Why is this optional now?

Reply:
In view port route config, they are not required, I realized this while dealing with viewports
```js
{ route: 'a', viewports: { a: {}, b: {} }
```

Reply:
moduleId is optional, route is not optional

Reply:
Yes, will need to change this. Probably because the way viewport typings is reused